lib/widgets/qpi/buttons.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
```

Comment:
move file to `lib/widgets/` for clarity; these buttons are used in almost all pages and not just by the qpi